src/opencl/image_function_opencl.cpp

```diff
@@ -1144,6 +1164,62 @@ namespace Image_Function_OpenCL
         multiCL::launchKernel2D( kernel, width, height );
     }
 
+    void SetPixel( Image & image, uint32_t x, uint32_t y, uint8_t value )
+    {
+        Image_Function::ParameterValidation( image );
+
+        const multiCL::OpenCLProgram & program = GetProgram();
+        multiCL::OpenCLKernel kernel( program, "setPixelOpenCL" );
+
+        if ( image.empty() || x >= image.width() || y >= image.height() )
+            throw imageException( "Bad input parameters in image function" );
+
+        const uint32_t width = image.width();
+        const uint32_t height = image.height();
+        const uint32_t rowSize = image.rowSize();
+        const uint32_t offset = x * rowSize + y;
+
+        kernel.setArgument( image.data(), offset, rowSize, width, height, value );
```

Comment:
You forgot to add `x` and `y` as parameters.

Reply:
added `x` and `y`. thanks